src/geocoder_control.js

```diff
@@ -7,6 +7,7 @@ var GeocoderControl = L.Control.extend({
     includes: L.Mixin.Events,
 
     options: {
+        proximity: false,
```

Comment:
I don't think this justifies a 3.0.0: it's not a breaking change, it's an improvement. This fits as a minor version bump.